slide-deck.org

```diff
@@ -2761,13 +2761,13 @@ Tag the variable(s) that contain the usernames you promised for /inventory/.
 #+END_SRC
 
 ** Create a file
-:PR
+:PR (these PR/ID/EN show up in the preso, you really want that?)
```

Comment:
That was supposed to be `:PROPERTIES:` and `:END:`. I did a visual column edit while headlines were folded and deleted a prefix, wacking text that was hidden!

Thought I had found all of them, thanks for pointing out.